python/pythonmonkey/__init__.py

```diff
@@ -5,13 +5,13 @@
 
 # Expose the package version
 import importlib.metadata
-__version__= importlib.metadata.version(__name__)
+__version__ = importlib.metadata.version(__name__)
 del importlib
 
 # Load the module by default to expose global APIs
-## builtin_modules
+# builtin_modules
 require("console")
 require("base64")
 require("timers")
 require("url")
-require("XMLHttpRequest")
\ No newline at end of file
+require("XMLHttpRequest")
```

Comment:
what changed here?

Reply:
autopep8 enforces a new line at the end of the file